.github/workflows/release-snapshot.yaml

```diff
@@ -0,0 +1,53 @@
+name: Release SNAPSHOT
+
+on:
+  push:
+    branches:
+      - master
+      - "[0-9]+"
+      - "[0-9]+.[0-9]+"
```

Comment:
We will release SNAPSHOT versions of release and support branches to properly refer them from each release and support branches from the `scalar` repository.